src/radical/saga/adaptors/cobalt/cobaltjob.py

```diff
@@ -271,22 +249,21 @@ def _cobaltscript_generator(url, logger, jd, ppn, is_cray=False, queue=None,
   #         % (number_of_processes, number_of_nodes, processes_per_host,
   #             (number_of_nodes * processes_per_host)), rse.NoSuccess, logger)
 
-    # Other funky math checks should go here ~
-
     # Set number of nodes
     cobalt_params += '#COBALT --nodecount %d\n' % number_of_nodes
 
     # Set the total number of processes
     cobalt_params += '#COBALT --proccount %d\n' % number_of_processes
 
-    # The Environments are added at the end because for now
-    # Cobalt isn't supporting spaces in the env variables...
-    # Which mess up the whole script if they are at the begining of the list...
+    cobalt_attrs = jd.system_architecture.get('options', [])
+    if cobalt_attrs:
+        cobalt_params += '#COBALT --attrs %s\n' % ':'.join(cobalt_attrs)
+
```

Comment:
Do you have an example of attributes intended to be passed this way?  Thanks!

Reply:
I didn't research it much, but have some that are in test script (`mcdram=cache:numa=quad:mig-mode=True`). 

I didn't add here a list of valid attributes (similar to Summit and Lassen), since here, besides attributes themselves, are values of that attributes, which could be more dynamic

Reply:
Thanks